com.aobuchow.themes.spectrum/plugin.xml

```diff
@@ -5,38 +5,24 @@
          point="org.eclipse.e4.ui.css.swt.theme">
       <theme
             basestylesheeturi="css/Spectrum.css"
-            id="modern.dark.theme.id"
```

Comment:
@AObuchow this change switched the theme to a light one on Windows. Remember https://github.com/AObuchow/Eclipse-Spectrum-Theme/issues/1#issuecomment-640028240 😉

Reply:
OOPS I totally forgot about that!!! Thank you for letting me know, I’ll fix that right away :)

Reply:
Fixed in https://github.com/AObuchow/Eclipse-Spectrum-Theme/commit/ee2cf5fd297f96e4d7f32c0fb41779fe781e20ce

Also: 100th commits wooo